README.md

```diff
@@ -310,6 +310,29 @@ clean = sanitizeHtml(dirty, {
 });
 ```
 
+### Allowed CSS Styles
+
+If you wish to allow specific CSS _styles_ on a particular element, you can do that with the `allowedStyles` option.
+Any other CSS classes are discarded.
+
+This implies that the `style` attribute is allowed on that element
+
+```javascript
+clean = sanitizeHtml(dirty, {
+        allowedTags: ['p'],
+        allowedAttributes: {
+          'p': ["style"],
+        },
+        allowedStyles: {
+          '*': {
+            'color': '*',
```

Comment:
can we allow users to specify a regexp? A wildcard defeats the point of having sanitization

Reply:
probably *do not* want to  allow a wildcard

Reply:
@pschwyter agreed, a wild card would introduce security concerns. That would enable anyone to enter malicious code

Reply:
RegExps would also enable whitelisting of measurement values. For example, if we wanted to whitelist pixels within font-size, all we'd need to do is append that to the regex.